code/modules/chemistry/tools/beakers.dm

```diff
@@ -254,9 +219,10 @@
 	desc = "You feel strangely warm and relaxed just looking at it."
 	icon = 'icons/misc/janstuff.dmi'
 	icon_state = "heartbottle"
-	icon_style = "heartbottle"
 	initial_volume = 50
 	initial_reagents = "love"
+	var/icon_style = "heartbottle"
+	var/image/fluid_image
```

Comment:
unused?

Reply:
This is Jan's secret ingredient flask, which was intentionally left unaltered. As a result, `fluid_image` is used; see `update_icon()` a few lines below.